02-2.py

```diff
@@ -0,0 +1,15 @@
+from utils import read_inputs
+
+inputs = read_inputs('inputs/input2.txt')
+
+right_pswd = 0
+
+for ele in inputs:
+	temp_tab = ele.split(' ')
+	tab_num = temp_tab[0].split('-')
+	letter_cnt = 0
```

Comment:
Same as before, we could have create a function that parse the line and return all the element in the right order ;)